### ISSUE TITLE
schema id ignored http://json-schema.org/draft-04/schema#

### ISSUE THREAD

**Issue by PlugIN73** [eslint/eslint#10892]
**Tell us about your environment**

* **ESLint Version:** v5.6.0
* **Node Version:** v10.9.0
* **npm Version:** 6.2.0

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```bash
root@bc1135588c5f:/usr/src/linter# node_modules/.bin/eslint
schema id ignored http://json-schema.org/draft-04/schema#
schema id ignored http://json-schema.org/draft-04/schema#
schema id ignored http://json-schema.org/draft-04/schema#
eslint [options] file.js [file.js] [dir]

Basic configuration:
...there are standard output...
```

**What did you expect to happen?**
Run without warnings.

**What actually happened? Please include the actual, raw output from ESLint.**
I see 3 warnings that schema id ignored. What is it? I run clear eslint, without any configuration.

**Comment by nzakas** [comment]
It looks like this warning might be coming from `ajv`.

@PlugIN73 Can you try running ESLint with a file? Either `node_modules/.bin/eslint test.js` or `npx eslint test.js` and see if you get the same output?

I'm not seeing this warning output when I try locally.
